vnext/Desktop.DLL/react-native-win32.oss.x64.def

```diff
@@ -17,7 +17,6 @@ EXPORTS
 ?GetConstants@ViewManagerBase@react@facebook@@UEBA?AUdynamic@folly@@XZ
 ?InitializeLogging@react@facebook@@YAX$$QEAV?$function@$$A6AXW4RCTLogLevel@react@facebook@@PEBD@Z@std@@@Z
 ?InitializeTracing@react@facebook@@YAXPEAUINativeTraceHandler@12@@Z
-?Make@IHttpResource@react@facebook@@SA?AV?$unique_ptr@UIHttpResource@react@facebook@@U?$default_delete@UIHttpResource@react@facebook@@@std@@@std@@XZ
```

Comment:
I think this was meant to be labeled with the Microsoft namespace like the ones below? We don't seem to need to export this, so I just removed it. I'm not sure OSS_RN was building for Win32 before this change, since I think we only build UWP in CI (I guess we'll find out).

Reply:
Yup, seems like win32 OSS_RN has just been slightly broken for a while. I will try to get that fixed up and add CI soon unless there's a reason we aren't running it.

Reply:
Yeah, we never supported really used win32 OSS_RN.  It was never added to CI, so its not surprising that it broke.

Reply:
Looks like we don't export this in the non-OSS def file, so removing is the right call here.